Changes

```diff
@@ -58,6 +58,10 @@ Working version
   (Stephen Dolan, report by Timothy Bourke, review by Mark Shinwell and Damien
   Doligez)
 
+- #13613: Functions from caml/skiplist.h and caml/lf_skiplist.h no longer raise
+  Out_of_memory exceptions that the runtime could not handle.
+  (Guillaume Munch-Maccagnoni, review by)
```

Comment:
the reviewer list should be updated by adding Stephen, and then this is good to go